### DISCUSSION TITLE
错过了购买时期

### DISCUSSION THREAD

**Discussion by EurekaMa** [Paladinfeng/MiaoProject#169] [General]
开发者大大辛苦了，之前一直在用TF版，错过了在应用市场购买的窗口期，现在测试结束了，市场下架了，生活不美好了。
开发者大大还会继续维护这个应用吗，如果再次上架的话我一定第一时间赶来**资持**

**Comment by Paladinfeng** [comment]
详情见TG公告 https://t.me/miaoproject

**Reply by EurekaMa** [reply]
求个预告，买爆

**Comment by sliencex** [comment]
在国内登陆TG真的比较困难，手机号码收验证码有问题。

**Reply by EurekaMa** [reply]
返场，等有消息了我在这里回复你

**Reply by sliencex** [reply]
好的，多谢呀，是不是必须是日区和美区的apple id？

**Reply by EurekaMa** [reply]
除国区应该都可以，提前充好票子

**Reply by EurekaMa** [reply]
> 好的，多谢呀，是不是必须是日区和美区的apple id？

今晚20:00 ~ 24:00

**Reply by polaire** [reply]
是说今晚20:00重新上架吗？请问app的名字是什么呀？

**Reply by EurekaMa** [reply]
> 是说今晚20:00重新上架吗？请问app的名字是什么呀？

miao project

**Reply by sliencex** [reply]
# 返场计划
优先看链接 https://paladinfeng.notion.site/140462eaa3134ff68909102cb2b37e32

## 返场时间（北京时间 GMT+8）：

- 2022/10/27    20:00 ~ 24:00
- 2022/10/28    20:00 ~ 24:00
- 2022/10/29    20:00 ~ 24:00
- 2022/10/30    0:00 ~ 24:00

## 登陆区域：

- 支持 Apple TV 的全区

## 还没有Apple TV / Apple TV 还在路上怎么办？

可以通过在苹果设备上打开链接跳转，链接地址  [https://apps.apple.com/us/app/miao-project/id1563875379](https://apps.apple.com/us/app/miao-project/id1563875379)

## 关于新激活方式的说明

1. 新激活限制允许 6 个 iCloud 激活
2. 注意⚠️：iCloud 账号是私人设备，共享 iCloud 非常危险，请不要共享或者登陆
3. 注意⚠️⚠️：请不要登陆其他人的 Apple ID 作为 iCloud 账号，别人可以远程锁定你的设备
4. 注意⚠️⚠️⚠️：登陆别人的 iCloud 账号，还会同步保存在 iCloud 中的B站信息

## 激活方法

1. 选择**远程连接**
2. 点击第一个输入框（请输入播放资源链接）
3. **手动**输入 **miao://platform/show**，注意要**手动**，复制粘贴可能在beta系统上无法触发
4. 注意：全部小写 / 英文标点符号
5. 重新启动应用

## 其他问题见使用说明：[https://github.com/Paladinfeng/MiaoProject/blob/master/HOW-TO-USE.md](https://github.com/Paladinfeng/MiaoProject/blob/master/HOW-TO-USE.md)

**Reply by polaire** [reply]
> # 返场计划
> 优先看链接 https://paladinfeng.notion.site/140462eaa3134ff68909102cb2b37e32
> 
> ## 返场时间（北京时间 GMT+8）：
> * 2022/10/27    20:00 ~ 24:00
> * 2022/10/28    20:00 ~ 24:00
> * 2022/10/29    20:00 ~ 24:00
> * 2022/10/30    0:00 ~ 24:00
> 
> ## 登陆区域：
> * 支持 Apple TV 的全区
> 
> ## 还没有Apple TV / Apple TV 还在路上怎么办？
> 可以通过在苹果设备上打开链接跳转，链接地址 https://apps.apple.com/us/app/miao-project/id1563875379
> 
> ## 关于新激活方式的说明
> 1. 新激活限制允许 6 个 iCloud 激活
> 2. 注意⚠️：iCloud 账号是私人设备，共享 iCloud 非常危险，请不要共享或者登陆
> 3. 注意⚠️⚠️：请不要登陆其他人的 Apple ID 作为 iCloud 账号，别人可以远程锁定你的设备
> 4. 注意⚠️⚠️⚠️：登陆别人的 iCloud 账号，还会同步保存在 iCloud 中的B站信息
> 
> ## 激活方法
> 1. 选择**远程连接**
> 2. 点击第一个输入框（请输入播放资源链接）
> 3. **手动**输入 **miao://platform/show**，注意要**手动**，复制粘贴可能在beta系统上无法触发
> 4. 注意：全部小写 / 英文标点符号
> 5. 重新启动应用
> 
> ## 其他问题见使用说明：https://github.com/Paladinfeng/MiaoProject/blob/master/HOW-TO-USE.md

感谢！

**Reply by polaire** [reply]
> > 是说今晚20:00重新上架吗？请问app的名字是什么呀？
> 
> miao project

感谢！

**Comment by polaire** [comment]
请问app价格多少啊？港服？

**Reply by sliencex** [reply]
估计7美元左右。

**Comment by wendao09** [comment]
> 请问app价格多少啊？港服？

48hkd
